src/main/java/domain/DailyAttendanceTime.java

```diff
@@ -0,0 +1,80 @@
+package domain;
+
+import static domain.AttendanceResult.ABSENCE;
+import static domain.AttendanceResult.ABSENCE_STANDARD_MINUTES;
+import static domain.AttendanceResult.ATTENDANCE;
+import static domain.AttendanceResult.LATE;
+import static domain.AttendanceResult.LATE_STANDARD_MINUTES;
+import static domain.AttendanceHistory.ABSENT_DEFAULT_HOUR;
+import static domain.AttendanceHistory.ABSENT_DEFAULT_MINUTE;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Arrays;
+import util.Convertor;
+
+public enum DailyAttendanceTime {
+    MONDAY(LocalTime.of(13, 0), DayOfWeek.MONDAY),
+    TUESDAY(LocalTime.of(10, 0), DayOfWeek.TUESDAY),
+    WEDNESDAY(LocalTime.of(10, 0), DayOfWeek.WEDNESDAY),
+    THURSDAY(LocalTime.of(10, 0), DayOfWeek.THURSDAY),
+    FRIDAY(LocalTime.of(10, 0), DayOfWeek.FRIDAY);
+
+    private final static int START_TIME = 8;
+    private final static int END_TIME = 23;
+    private final LocalTime attendanceTime;
+    private final DayOfWeek dayOfWeek;
+
+    DailyAttendanceTime(LocalTime attendanceTime, DayOfWeek dayOfWeek) {
+        this.attendanceTime = attendanceTime;
+        this.dayOfWeek = dayOfWeek;
+    }
+
+    public static AttendanceResult getDailyAttendanceResult(LocalDateTime attendanceTime) {
+        Holiday.validateHoliday(attendanceTime);
+        validateOpeningHours(attendanceTime);
+        DailyAttendanceTime dailyAttendanceTime = findDailyAttendanceTime(attendanceTime);
+        return getAttendanceResult(dailyAttendanceTime, attendanceTime);
+    }
+
+    public static boolean canAttendance(LocalDate attendanceDate) {
+        boolean canAttendance = Arrays.stream(DailyAttendanceTime.values())
+                .anyMatch(dailyAttendanceTime -> dailyAttendanceTime.dayOfWeek == attendanceDate.getDayOfWeek());
+        return canAttendance && !Holiday.isHoliday(attendanceDate);
+    }
+
+    private static DailyAttendanceTime findDailyAttendanceTime(LocalDateTime attendanceTime) {
+        return Arrays.stream(DailyAttendanceTime.values())
+                .filter(day -> attendanceTime.getDayOfWeek() == day.dayOfWeek)
+                .findAny()
+                .orElseThrow(() -> new IllegalArgumentException(
+                        String.format("[ERROR] %s은 등교일이 아닙니다.", Convertor.dateFormattingForInput(attendanceTime))
+                ));
+    }
+
+    private static AttendanceResult getAttendanceResult(DailyAttendanceTime dailyAttendanceTime,
+                                                        LocalDateTime attendanceTime) {
+        LocalTime todayAttendanceTime = dailyAttendanceTime.attendanceTime;
+        LocalTime attendanceLocalTime = attendanceTime.toLocalTime();
+        if (attendanceLocalTime.isAfter(todayAttendanceTime.plusMinutes(ABSENCE_STANDARD_MINUTES))) {
```

Comment:
사소하지만 Duration을 활용해서 분을 체크하는 방식도 가능하겠네요. 😃 (이 부분은 꼭 수정해달라는 것이 아니고 Duration을 활용하는 방식도 있다 정도로 생각해주시면 될 것 같아요)

```suggestion
        if (Duration.between(attendanceLocalTime, todayAttendanceTime).toMinutes() > ABSENCE_STANDARD_MINUTES) {
```

Reply:
헉! Duration은 처음봅니다! 좋은 정보 감사합니다! 수정하면서 공부해보고 싶네요 ㅎㅎ